internal/test/e2e/e2e_test.go

```diff
@@ -345,4 +345,39 @@ func runAllTests(t *testing.T, kubectl *kubectlContext) {
 		}, time.Minute*2, time.Second*10)
 		require.NoError(t, err)
 	})
+
+	t.Run("ScaleUp", func(t *testing.T) {
+		// Attempt to scale up to five nodes
+		err = kubectl.Scale("etcdcluster/my-cluster", 5)
+		require.NoError(t, err)
+
+		etcdClient, err := etcd.New(etcdConfig)
+		require.NoError(t, err)
+
+		err = try.Eventually(func() error {
+			t.Log("Listing etcd members")
+			members, err := etcd.NewMembersAPI(etcdClient).List(ctx)
+			if err != nil {
+				return err
+			}
+
+			if len(members) != 5 {
+				return errors.New(fmt.Sprintf("expected %d etcd peers, got %d", 5, len(members)))
+			}
+
+			for _, member := range members {
+				if len(member.ClientURLs) == 0 {
+					return errors.New("peer has no client URLs")
+				}
+				if len(member.PeerURLs) == 0 {
+					return errors.New("peer has no peer URLs")
+				}
```

Comment:
More specifically, wouldn't we expect them all to have 4 peer URLS (or 5, if the peer's own url is also included)

Reply:
It only includes the URLs for *that peer*

```
$ etcdctl member list
19838fa9c1e93efc: name=my-cluster-1 peerURLs=http://my-cluster-1.my-cluster.default.svc:2380 clientURLs=http://my-cluster-1.my-cluster.default.svc:2379 isLeader=false
9b06489270dfecaf: name=my-cluster-4 peerURLs=http://my-cluster-4.my-cluster.default.svc:2380 clientURLs=http://my-cluster-4.my-cluster.default.svc:2379 isLeader=false
bc96187fb93967e3: name=my-cluster-3 peerURLs=http://my-cluster-3.my-cluster.default.svc:2380 clientURLs=http://my-cluster-3.my-cluster.default.svc:2379 isLeader=false
cb4790e30037cb3e: name=my-cluster-2 peerURLs=http://my-cluster-2.my-cluster.default.svc:2380 clientURLs=http://my-cluster-2.my-cluster.default.svc:2379 isLeader=true
f9fb3409c9a0b367: name=my-cluster-0 peerURLs=http://my-cluster-0.my-cluster.default.svc:2380 clientURLs=http://my-cluster-0.my-cluster.default.svc:2379 isLeader=false
```